packages/blade/src/components/Chip/ChipGroup.stories.tsx

```diff
@@ -727,3 +727,80 @@ chipRef.parameters = {
     exclude: ['icon'],
   },
 };
+
+export const MultiChipStory: StoryFn<typeof ChipGroupComponent> = () => {
+  const chipArray = [
+    {
+      value: '100',
+      label: '₹100',
+    },
+    {
+      value: '500',
+      label: '₹500',
+    },
+    {
+      value: '1000',
+      label: '₹1000',
+    },
+    {
+      value: '2000',
+      label: '₹2000',
+    },
+    {
+      value: '5000',
+      label: '₹5000',
+    },
+    {
+      value: '10000',
+      label: '₹10000',
+    },
+    {
+      value: '20000',
+      label: '₹20000',
+    },
+    {
+      value: '50000',
+      label: '₹50000',
+    },
+    {
+      value: '100000',
+      label: '₹100000',
+    },
+    {
+      value: '200000',
+      label: '₹200000',
+    },
+  ];
+
+  return (
+    <Box gap="spacing.3" display="flex" flexDirection="column">
+      <ChipGroupComponent
+        selectionType="single"
+        label="Select a gift card with value (without custom chipGroupContainerLayout)"
+      >
+        {chipArray.map((chip, index) => (
+          <ChipComponent key={index} value={chip.value}>
+            {chip.label}
+          </ChipComponent>
+        ))}
+      </ChipGroupComponent>
+      <ChipGroupComponent
+        selectionType="single"
+        label="Select a gift card with value (custom chipGroupContainerLayout)"
+      >
+        <Box
+          display="grid"
+          gridTemplateColumns="repeat(3, minmax(0,120px))"
+          gridTemplateRows="repeat(3, minmax(0,30px))"
+          gap="spacing.3"
+        >
+          {chipArray.map((chip, index) => (
+            <ChipComponent key={index} value={chip.value} width="120px">
```

Comment:
🤔 We shouldn't need to add explicit pixel values in ChipComponent. 

`<ChipComponent key={index} value={chip.value} width="100%" />`

I would assume this would work and the chip should automatically stretch/flex to available space. 


Not to mention explicit pixel values will break the layout if there is no space: 

<img width="281" alt="image" src="https://github.com/user-attachments/assets/56cb7daa-3d3b-4341-bb51-30537d114ff7" />